build.py

```diff
@@ -112,6 +153,32 @@ def _parse_args():
         help="Path to the Android NDK. Typically `<Android SDK>/ndk/<ndk_version>`.",
     )
 
+    # iOS build options
+    parser.add_argument("--ios", action="store_true", help="build for ios")
```

Comment:
could we put all the cross compiling options (`ios`, `android`, and whatever other ones may be added) into a mutually exclusive group? I assume we won't support cross-compiling for more than one platform in one build.py call.